github_search/lib/entity/github_repository.dart

```diff
@@ -27,24 +32,39 @@ class GitHubRepository with _$GitHubRepository {
 }
 
 extension GitHubRepositoryExtension on GitHubRepository {
-  String toWithSeparated(int value) {
-    final formatter = NumberFormat("#,###");
-    return formatter.format(value);
-  }
-
   String get starCountWithComma {
-    return toWithSeparated(starCount);
+    return _toWithSeparated(starCount);
   }
 
   String get watchCountWithComma {
-    return toWithSeparated(watchCount);
+    return _toWithSeparated(watchCount);
   }
 
   String get forkCountWithComma {
-    return toWithSeparated(forkCount);
+    return _toWithSeparated(forkCount);
   }
 
   String get issueCountWithComma {
-    return toWithSeparated(issueCount);
+    return _toWithSeparated(issueCount);
+  }
+}
+
+@freezed
+class GitHubGetRepository with _$GitHubGetRepository {
+  const factory GitHubGetRepository({
+    @JsonKey(name: 'subscribers_count') required int subscribersCount
+  }) = _GitHubGetRepository;
+
+  factory GitHubGetRepository.fromJson(Map<String, dynamic> json) => _$GitHubGetRepositoryFromJson(json);
+}
+
+extension GitHubGetRepositoryExtension on GitHubGetRepository {
+  String get subscribersCountWithComma {
+    return _toWithSeparated(subscribersCount);
   }
+}
+
+String _toWithSeparated(int value) {
```

Comment:
this method name is bad. it's difficult to know its mean.

idea is ;
String _toWithSeparatedComma(int value) {}

Reply:
fixed: https://github.com/shusuke0812/flutter-engineer-codecheck/pull/5/commits/5dfe8ab355757feaa19761e918c1a9b8fc8ede93